resume.yaml

```diff
@@ -1,121 +0,0 @@
-name: Sean Gatewood
-contact:
-  email: stg2bd@virginia.edu
-  linkedin: linkedin.com/in/sean-t-gatewood
-  github: github.com/sgatewood
-education:
-  school: University of Virginia
-  school titles:
-    - Rodman Scholar
-    - Raven Society
-  degree: B.S. Computer Science
-  gpa: 3.95
-  graduation: May 2020
-  notable courses:
-    - Computer Graphics
-    - Algorithms
-    - Advanced Software Development
-    - Introduction to Cybersecurity
-    - Software Testing
-experience:
-  - title: Software Engineer II
-    company: Appian Corporation
-    location: McLean, VA
-    dates: August 2020 - present
-    notes:
-      - Currently work on a SCRUM team that creates and maintains a variety of internal developer tooling and infrastructure.
-      - Helped design and implement a modern feature toggles framework for a legacy codebase. This project was pretty extensive and incorporated a variety of tooling and services, including the creation of standalone services with Kubernetes. 
```

Comment:
Sell the extensiveness and framework creation: maybe something like: 

```
Designed and implemented a modern feature toggle framework for a legacy monolith codebase
- Created standalone Kubernetes services and <idk wrote docs, did maintenance/support/whatever>
- Saved X (guess) hours of developer time OR Reduced # of SEV 0s (or whatever you call blocker bugs) by <your best guess>
```

IMO It's ok to write "designed and implemented" when you were a contributor / co-author. It reads more powerfully, and everyone knows that no software at a large org is made in isolation (hopefully 😆), so no one is gonna think you're trying to steal all the credit or anything like that